### PR TITLE
fix(fmt): auto formatted flux files in stdlib/testing using flux cli

### PR DESCRIPTION
**Pull request by skartikey**
Formatted flux files in stdlib/testing folder using flux cli (./flux fmt -w  stdlib/testing)

### CODE REVIEW

File: stdlib/testing/chronograf/aggregate_window_count_test.flux [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package chronograf_test
- 
+
```
Comment (rockstar) [inline stdlib/testing/chronograf/aggregate_window_count_test.flux:2]: Love this.

File: stdlib/testing/chronograf/aggregate_window_count_test.flux [hunk 2]
```diff
@@ -68,7 +69,6 @@ inData = "
 ,,1,2018-05-22T00:00:40Z,mem,percentage,host.remote,35
 ,,1,2018-05-22T00:00:50Z,mem,percentage,host.remote,35
 "
-
```
Comment (rockstar) [inline stdlib/testing/chronograf/aggregate_window_count_test.flux:71]: Probably fine with this, though we often add this whitespace for reasons. I _kinda_ wish the formatter just left it alone, but I don't want to scope creep this patch. Maybe an issue is warranted?

File: stdlib/testing/influxql/aggregate_group_by_time_test.flux [hunk 3]
```diff
@@ -324,16 +324,16 @@ outData = "
...
-	|> aggregateWindow(every: 5h, fn: sum, timeSrc: "_start")
-	|> rename(columns: {_time: "time", _value: "sum"})
-	|> drop(columns: ["_field", "_start", "_stop"])
+    |> range(start: 1970-01-01T00:00:00Z, stop: 1970-01-01T20:00:00.000000001Z)
+    |> filter(fn: (r) => r._measurement == "m")
+    |> filter(fn: (r) => r._field == "f")
+    |> group(columns: ["_measurement", "_field"])
+    |> aggregateWindow(every: 5h, fn: sum, timeSrc: "_start")
+    |> rename(columns: {_time: "time", _value: "sum"})
+    |> drop(columns: ["_field", "_start", "_stop"])
```
Comment (rockstar) [inline stdlib/testing/influxql/aggregate_group_by_time_test.flux:333]: Neat. 

*whispers*Spaces are better.

File: stdlib/testing/kapacitor/combine_pivot_test.flux [hunk 4]
```diff
@@ -1,8 +1,9 @@
 package testdata_test
 
+
 import "testing"
 
-option now = () => (2030-01-01T00:00:00Z)
+option now = () => 2030-01-01T00:00:00Z
```
Comment (rockstar) [inline stdlib/testing/kapacitor/combine_pivot_test.flux:6]: I am loving this change.

File: stdlib/testing/kapacitor/combine_pivot_test.flux [hunk 5]
```diff
@@ -35,21 +35,16 @@ outData = "
 ,,0,memory,2018-12-19T22:14:10Z,6,9
 ,,0,memory,2018-12-19T22:14:20Z,3,8
 "
+t_combine_join = (table=<-) => table
+    |> range(start: 2018-12-15T00:00:00Z)
+    |> drop(columns: ["_start", "_stop"])
+    |> pivot(rowKey: ["_time", "_measurement"], columnKey: ["_field"], valueColumn: "_value")
```
Comment (rockstar) [inline stdlib/testing/kapacitor/combine_pivot_test.flux:41]: Love this update.

File: stdlib/testing/kapacitor/combine_pivot_test.flux [hunk 6]
```diff
@@ -35,21 +35,16 @@ outData = "
...
-    (table
-        |> range(start: 2018-12-15T00:00:00Z)
-        |> drop(columns: ["_start", "_stop"])
-		|> pivot(rowKey:["_time", "_measurement"], columnKey: ["_field"], valueColumn: "_value"))
-
-test _combine_join = () =>
-	({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_combine_join})
-
-
+test _combine_join = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_combine_join})
```
Comment (rockstar) [inline stdlib/testing/kapacitor/combine_pivot_test.flux:43]: Not sure how I feel about this one, but I get it.

File: stdlib/testing/kapacitor/combine_pivot_test.flux [hunk 7]
```diff
@@ -35,21 +35,16 @@ outData = "
...
 // Equivalent TICKscript query:
 //
 // stream
 //    |from()
 //      .measurement('request_latency')
 //    |combine(lambda: "_field"=='user1;, lambda: "_field"=='user2')
-//      .as('user1', 'user2')
\ No newline at end of file
+//      .as('user1', 'user2')
```
Comment (rockstar) [inline stdlib/testing/kapacitor/combine_pivot_test.flux:50]: Whether a project/language/file has newlines at the end of files is one of the greatest annoyances of my entire career, so I'm glad to see our formatter being opinionated here.

File: stdlib/testing/kapacitor/eval_map_with_test.flux [hunk 8]
```diff
@@ -23,17 +23,13 @@ outData = "
...
-	({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_map})
-
+test _map = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_map})
 // Equivalent TICKscript query:
 // stream 
-// 	 |eval(lambda: 2 * _value)
-//		.as('_newValue')
\ No newline at end of file
+//  |eval(lambda: 2 * _value)
+ //.as('_newValue')
```
Comment (rockstar) [inline stdlib/testing/kapacitor/eval_map_with_test.flux:35]: This is weird. I think this might be a legitimate bug. We shouldn't be indenting comments, and we should preserve whatever whitespace they have.

File: stdlib/testing/pandas/extract_regexp_findString_test.flux [hunk 9]
```diff
@@ -31,14 +31,9 @@ outData = "
...
-t_string_extract = (table=<-) =>
-	(table
-		|> range(start: 2018-05-22T19:53:26Z)
-		|> map(fn: (r) => ({r with extract: regexp.findString(r: re, v: r._value)}))
-	)
-
-test _string_extract = () =>
-         ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_string_extract})
\ No newline at end of file
+test _string_extract = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_string_extract})
```
Comment (rockstar) [inline stdlib/testing/pandas/extract_regexp_findString_test.flux:39]: Now that I've read this a few times, I take it back. I like the `test` being read like this.

**Review by rockstar — APPROVED** [review]
An exciting anecdote: when I first starting reading this newly formatted flux code, I wasn't sure how I  felt about it. As I got farther in the patch, my brain had already created patterns to read it faster, which is quite the testament to having opinionated formatting.

I'd like to have a conversation and maybe file a couple of issues about some formatting, but nothing blocking a good merge here.

File: stdlib/testing/kapacitor/eval_map_with_test.flux (hunk 8 above)
Comment (nathanielc) [reply stdlib/testing/kapacitor/eval_map_with_test.flux:35]: Agreed, let's create an issue for this one.

**Review by nathanielc — APPROVED** [review]
I also took a quick look and love it. The consistency is beautiful. Thanks
